Loop/Views/StatusTableView.swift

```diff
@@ -194,15 +194,16 @@ struct StatusTableView: View {
     
     func isDisabled(action: ToolbarAction) -> Bool {
         switch action {
-        case .addCarbs, .bolus, .settings:
+        case .addCarbs, .bolus, .settings, .presets:
             false
-        case .presets:
-            !viewModel.onboardingManager.isComplete
         }
     }
     
     var body: some View {
         wrappedView
+            .onChange(of: viewModel.settingsViewModel.presetsViewModel.activePreset) { _, _ in
+                viewController.redrawCharts()
```

Comment:
wondering if this should call `reloadData(animated: )` instead of making `redrawCharts()` internal and calling that function.

Reply:
I can do that. The only thing that needs a reload is the charts but transitioning the func from `private` to `internal` just for this 1 use case maybe isn't necessary.